src/taipy/rest/api/resources/datanode.py

```diff
@@ -143,12 +173,22 @@ class DataNodeList(Resource):
       tags:
         - api
       summary: Create a datanode
-      description: Create a new datanode
-      requestBody:
-        content:
-          application/json:
-            schema:
-              DataNodeSchema
+      description: |
+        Create a data node from its config_id. If the config does not exist, a 404 error is returned.
+
+        When the authorization feature is activated (available in the **Enterprise** edition only), this endpoint requires _TAIPY_EDITOR_ role.
+
+        Code example:
+
+        ```shell
+          curl -X POST http://localhost:5000/api/v1/datanodes?config_id=CONFIG_ID
+        ```
+      parameters:
+        - in: query
+          name: config_id
+          schema:
+            type: string
+          description: The id of the data node config
```

Comment:
'The id of the data node config'
->
'The identifier of the data node configuration.'